lib/active_fedora/nested_attributes.rb

```diff
@@ -181,8 +183,8 @@ def raise_nested_attributes_record_not_found(association_name, record_id)
       raise ObjectNotFoundError, "Couldn't find #{reflection.klass.name} with ID=#{record_id} for #{self.class.name} with ID=#{id}"
     end
 
-    def call_reject_if(association_name, attributes)
-      return false if has_destroy_flag?(attributes)
+    def call_reject_if(association_name, attributes, allow_destroy)
```

Comment:
Right, I think we should pull this code in: https://github.com/rails/rails/blob/master/activerecord/lib/active_record/nested_attributes.rb#L483